packages/@react-spectrum/breadcrumbs/src/BreadcrumbItem.tsx

```diff
@@ -11,6 +11,8 @@
  */
 
 import {BreadcrumbItemProps} from '@react-types/breadcrumbs';
+// eslint-disable-next-line monorepo/no-internal-import
```

Comment:
something to do with alpha package? we should bump it to release

Can't do the UI icon replacement map because chevron right is used in too many places and I only wanted to adjust the one for breadcrumbs